generators/server/templates/dotnetcore/src/Project.Domain/Entities/AuditedEntityBase.cs.ejs

```diff
@@ -0,0 +1,25 @@
+<%#
+ Copyright 2013-2020 the original author or authors from the JHipster project.
+ This file is part of the JHipster project, see https://www.jhipster.tech/
+ for more information.
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-%>
+using System;
+using <%= namespace %>.Domain.Interfaces;
+
+namespace <%= namespace %>.Domain {
+    public class AuditedEntityBase : IAuditedEntityBase {
```

Comment:
Maybe make this class abstract since it will only be used by inheriting from it

Reply:
abstract here is good idea

Reply:
@maznag can you apply the change ?